src/pre_commit_terraform/_cli.py

```diff
@@ -28,8 +26,7 @@ def invoke_cli_app(cli_args: list[str]) -> ReturnCodeType:
         raise
     except PreCommitTerraformRuntimeError as unhandled_exc:
         print(
-            f'App execution took an unexpected turn: {unhandled_exc !s}. '
-            'Exiting...',
+            f'App execution took an unexpected turn: {unhandled_exc !s}. ' 'Exiting...',
```

Comment:
This folding looks incorrect
```suggestion
            f'App execution took an unexpected turn: {unhandled_exc !s}. Exiting...',
```

Reply:
This is how the formatters are set up in Max's branch, it's not something that's a part of the PR originally.